docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,82 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+You can deploy the monitoring stack shipped with lokomotive as shown in the Prometheus Operator component [guide](../configuration-reference/components/prometheus-operator.md).
+
+## Prerequisites
+
+- Prometheus Operator component is deployed in `monitoring` namespace. If you have specified a different namespace in config then change the commands accordingly, `kubectl -n <your namespace> ...`.
+
+## Prometheus
+
+Execute the command to port forward Prometheus locally on port `9090`:
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
+```
+
+Now open the following URL: [http://localhost:9090](http://localhost:9090).
+
+## Alertmanager
+
+Run following command to port forward Alertmanager locally on port `9093`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093:9093
+```
+
+Now open the following URL: [http://localhost:9093](http://localhost:9093).
+
+## Grafana
+
+### Using Port Forward
+
+Run following command to port forward Grafana dashboard locally on port `8080`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-grafana 8080:80
+```
+
+Obtain Grafana `admin` user password by running following command:
+
+```
+kubectl -n monitoring get secret prometheus-operator-grafana -o jsonpath='{.data.admin-password}' | base64 -d && echo
+```
+
+Now open the following URL: [http://localhost:8080](http://localhost:8080). Enter username `admin` and password obtained from previous step.
+
+### Using Ingress
+
+Make sure you are using a configuration for Grafana in Prometheus Operator something similar to following:
+
+```tf
+component "prometheus-operator" {
+  grafana {
+    ingress {
+      host = "grafana.mydomain.com"
+    }
+  }
+}
+```
+
+**NOTE**: If you are running this component on Packet then make sure that you have made a DNS entry for `grafana.mydomain.com` against the Packet EIP.
```

Comment:
I feel we should ensure Packet isn't a "special case" rather than mention it as a special case here. Maybe it's time to address https://github.com/kinvolk/lokomotive/issues/470?

Reply:
I can put the information about AWS as well. Like it is automatically available, just add the ingress host as `grafana.<cluster name>.<domain>`.

Reply:
Again, IMO Packet should have a similar UX to AWS. But I understand that treating the feature parity problems is out of scope for this PR.

Reply:
This is modified, PTAL.